infra/helper.py

```diff
@@ -153,6 +153,9 @@ def get_parser():  # pylint: disable=too-many-statements
   build_fuzzers_parser.add_argument('source_path',
                                     help='path of local source',
                                     nargs='?')
+  build_fuzzers_parser.add_argument('--mount_location',
```

Comment:
let's rename this to `mount_path` to be consistent with `source_path`, since it's expected to be used together.

Reply:
I renamed both the flag and the argument to `mount_path` for consistency.